src/transformers/agents/tools.py

```diff
@@ -441,8 +441,8 @@ def compile_jinja_template(template):
     except ImportError:
         raise ImportError("template requires jinja2 to be installed.")
 
-    if version.parse(jinja2.__version__) <= version.parse("3.0.0"):
-        raise ImportError("template requires jinja2>=3.0.0 to be installed. Your version is " f"{jinja2.__version__}.")
+    if version.parse(jinja2.__version__) < version.parse("3.1.0"):
```

Comment:
Does this match with jinja specified in the library e.g. setup.py?

Reply:
We don't actually make Jinja a library requirement - chat templates were a somewhat peripheral feature, so I didn't want to add an forced dependency for them at first, plus they're already a dependency for `torch`.

However, given that they're becoming a lot more core, maybe I should consider doing that in a separate PR!